b2b-sample/guardio-insurance-sample-application/components/common/logo/logo.js

```diff
@@ -22,31 +22,49 @@ import logoImage from "../../../public/logo.png";
 
 /**
  * 
- * @param prop - imageSize (small, medium, large or x-large)
+ * @param prop - imageSize `small` | `medium` | `large` | `x-large`
  * 
  * @returns Logo component
  */
 export default function Logo(prop) {
 
-    const { imageSize } = prop;
+    const { imageSize, white } = prop;
     
-    function switchImageSize(size) {
+    const getImageStyle = (size, white) => {
+
+        let imageStyle = {};
+
         switch (size) {
             case "small":
-                return { width: "200px" };
+                imageStyle["width"] = "200px" ;
+
+                break;
             case "medium":
-                return { width: "250px" };
+                imageStyle["width"] = "350px" ;
+
+                break;
             case "large":
-                return { width: "600px" };
+                imageStyle["width"] = "600px" ;
+
+                break;
             case "x-large":
-                return { width: "600px" };
+                imageStyle["width"] = "600px" ;
+
+                break;
             default:
                 break;
         }
-    }
+
```

Comment:
Let's remove redundant new lines. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/acb244b9a8bb35ce524de853271162c3ffa83c4d